packages/frontend-2/components/settings/workspaces/members/GuestsTable.vue

```diff
@@ -84,6 +84,17 @@
       :user="userToModify"
       :workspace-id="workspaceId"
     />
+
+    <SettingsSharedChangeRoleDialog
+      v-model:open="showChangeUserRoleDialog"
+      :name="userToModify?.user.name ?? ''"
```

Comment:
I think this prop is actually redundant, seems it's not used in this component anymore, let's remove it